components/raftstore/src/store/fsm/apply.rs

```diff
@@ -336,6 +340,7 @@ impl ApplyContext {
             sync_log_hint: false,
             exec_ctx: None,
             use_delete_range: cfg.use_delete_range,
+            enable_multi_batch_write: opt.is_enable_multi_batch_write(),
```

Comment:
It looks like this should instead be `engines.kv.c().support_write_batch_vec`. Otherwise `support_write_batch_vec` is unused.

Reply:
Ok.